app/views/layouts/_footer_duo.html.erb

```diff
@@ -0,0 +1,20 @@
+<footer class="grid grid-cols-4 grid-rows-1 gap-0 text-sm text-gray-700 p-4">
```

Comment:
This seems not a good route to go. i believe we are missing a few items in the turbo stuffs.